### PR TITLE
Thunderbird support in PollBot

### LINKED ISSUE #51: Add support for Thunderbird

### PR DESCRIPTION
**Pull request by jfx2006**
Resolves #51 

Proof-of-concept addition of a Thunderbird product to PollBot. State: functional

I plan to do some cleanup,and try to make the changes themselves more consistent in style. I'm sure some tests will fail too.

### CODE REVIEW

File: pollbot/tasks/archives.py [hunk 1]
```diff
@@ -23,22 +31,73 @@
...
+        url = "https://hg.mozilla.org/{}/raw-file/tip/{}/locales/all-locales".format(
+            repo_name, locales_path
+        )
     elif channel in (Channel.BETA, Channel.AURORA):
-        url = ("https://hg.mozilla.org/releases/mozilla-beta/raw-file/{}/"
-               "browser/locales/shipped-locales").format(tag)
+        url = ("https://hg.mozilla.org/releases/{}/raw-file/{}/"
+               "{}/locales/shipped-locales").format(repo_name, tag, locales_path)
     elif channel is Channel.RELEASE:
-        url = ("https://hg.mozilla.org/releases/mozilla-release/raw-file/{}/"
```
Comment (mozbhearsum) [inline pollbot/tasks/archives.py]: It looks like this can be combined with the previous block now that the repo handling has been centralized.

**Comment by jfx2006** [comment]
https://github.com/thundernest/thunderbird-website/pull/175 adds support to thunderbird.net to easily check download links. I'll tackle the failing tests next weekend probably.

**Comment by mozbhearsum** [comment]
@jfx2006 - let me know when this is ready for a full review (I see it's still in draft form).

**Comment by mozbhearsum** [comment]
Just noticed I didn't get notified when you marked this as non-draft, oops. @jfx2006 - I assume this is still ready as far as you're concerned? Sorry this sat for so long.

**Comment by jfx2006** [comment]
> Just noticed I didn't get notified when you marked this as non-draft, oops. @jfx2006 - I assume this is still ready as far as you're concerned? Sorry this sat for so long.

Yes it is. Thanks. 🙂

**Review by mozbhearsum — APPROVED** [review]
Let's do it!

**Pull request merged**